### PR TITLE
Sga 46 localization support

### PR DESCRIPTION
**Pull request by mantasjasikenas**
## PR Checklist

- [x] Added language selector in settings
- [x] Translated text to Lithuanian
- [x] Added Information screen in avatar dropdown menu

### CODE REVIEW

File: app/src/main/res/values-lt/strings.xml [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    <string name="awesome">Nuostabu!</string>
+    <string name="top_5_users_in_the_global_leaderboard">Top 5 vartotojai pasaulinėje lyderių lentelėje</string>
+    <string name="other_users">Kiti vartotojai</string>
+    <string name="other_users_in_the_global_leaderboard">Kiti vartotojai pasaulinėje lyderių lentelėje</string>
+    <string name="no_credentials_found_please_add_a_google_account_to_your_device">Nerasta prisijungimo duomenų. Prašome pridėti Google paskyrą į savo įrenginį.</string>
+    <string name="create_exercise_session">Sukurti treniruotės sesiją</string>
+    <string name="exercise_session">Treniruotės sesija</string>
+    <string name="exercise_session_in_progress">Vyksta treniruotės sesija</string>
+    <string name="profile_label">Paskyra</string>
+    <string name="profile">Paskyra</string>
```
Comment (paulius-mongirdas) [inline app/src/main/res/values-lt/strings.xml]: Duplicate values

File: app/src/main/java/com/github/k409/fitflow/ui/screen/level/LevelScreen.kt
Comment (paulius-mongirdas) [inline app/src/main/java/com/github/k409/fitflow/ui/screen/level/LevelScreen.kt:1]: LevelCard does not translate it's reward. Modify LevelCard.kt file where reward description is assigned.

File: app/src/main/java/com/github/k409/fitflow/ui/screen/settings/SettingsScreen.kt
Comment (paulius-mongirdas) [inline app/src/main/java/com/github/k409/fitflow/ui/screen/settings/SettingsScreen.kt:1]: SettingsScreen contents are not translated when language is changed

File: app/src/main/res/values-lt/strings.xml [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+    <string name="has_been_added_to_your_inventory">buvo įtrauktas į jūsų inventorių</string>
+    <string name="has_been_sold">buvo parduotas</string>
+    <string name="hydration_notification_progress">Šios dienos pažanga:</string>
+    <string name="today__progress_liters">Šiandienos pažanga yra %1$s/%2$s litrų</string>
+    <string name="default_notification_channel_id">Numatytas</string>
+    <string name="you">Tu</string>
+    <string name="progress_tab_title">PROGRESAS</string>
+    <string name="activities_tab_title">VEIKLA</string>
+    <string name="global_leaderboard_tab_title">LYGŲ LENTELĖ</string>
+    <string name="current_month_progress">Einamojo mėnesio eiga</string>
```
Comment (paulius-mongirdas) [inline app/src/main/res/values-lt/strings.xml]: "LYGŲ LENTELĖ" is misleading and could be replaced with a more accurate translation. Maybe something like "LYDERIŲ LENTELĖ"

File: app/src/main/res/values-lt/strings.xml [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+    <string name="cancel">Atšaukti</string>
+    <string name="enter_your_name_here">Įveskite čia savo vardą</string>
+    <string name="select_your_age">Pasirinkite savo amžių</string>
+    <string name="select">Pasirinkite</string>
+    <string name="select_your_gender">Pasirinkite savo lytį</string>
+    <string name="select_your_weight">Pasirinkite savo svorį</string>
+    <string name="select_your_height">Pasirinkite savo ūgį</string>
+    <string name="something_went_wrong_try_again">Kažkas nutiko. Bandyk iš naujo</string>
+    <string name="Yesterday">vakar</string>
+    <string name="Month">Šį mėnesį</string>
```
Comment (paulius-mongirdas) [inline app/src/main/res/values-lt/strings.xml]: vakar should be uppercase here

**Review by paulius-mongirdas — CHANGES_REQUESTED** [review]

**Comment by paulius-mongirdas** [comment]
Maybe remove option for "System" language, because it does not work if system language is not English or Lithuanian

**Review by paulius-mongirdas — APPROVED** [review]

**Review by GillWell — APPROVED** [review]

**Pull request merged**